### ISSUE TITLE
[2021.04.12]

### ISSUE THREAD

**Issue by yeonnwoo** [Yeonwoo-Kim/Traffic-Analysis-System-by-Time#8]
## 현재까지의 진행사항

* #### Test data 총 1484장 labelImg를 이용해 GT 찍기 완료
* #### xml type의 GT 정보 - yolo format으로 변경 코드 작업 완료
* #### 연구실 서버와 원격 연결 완료
* #### 1차 성능 평가를 위해 원격 서버로 YOLOv3 학습 돌리는 중

![KakaoTalk_20210413_104014721 (1)](https://user-images.githubusercontent.com/50096655/114484614-84d28e00-9c45-11eb-9879-fd8ca7cae6f4.gif)

## TODO

* #### 학습된 weight 가져와서 test data or real-time으로 test해보기
* #### 임베디드 카메라 관련 논의 필요 - 기존의 카메라 사용? or 새로운 것 사용
* #### 최적화 관련 논의 필요

**Comment by suzyrhkr** [comment]
### *** YOLO format(pytorch) -> (class x y w h) 
* train (10032 장)
-images
-labels 
* test (1428 장)
-images
-labels 

### *** QAT fine-tuning -> (x_min,y_min,x_max,y_max,class)
* train (10032 장)
-images
-labels 
* test (1428 장)
-images
-labels 

두 포맷 작업 완료
